Code/Framework/AzCore/AzCore/Component/ComponentApplicationBus.h

```diff
@@ -211,3 +211,5 @@ namespace AZ
     //! Used by components to make requests of the component application.
     using ComponentApplicationBus = AZ::EBus<ComponentApplicationRequests, ComponentApplicationRequestsEBusTraits>;
 }
+
+DECLARE_EBUS_EXTERN(ComponentApplicationRequests, ComponentApplicationRequestsEBusTraits);
```

Comment:
nit: I'm not sure if it's GitHub's viewer,  but it seems that a few of the files like this one are missing newlines at the end.